clients/cobol-lsp-vscode-extension/src/services/ProcessorGroupsLoader.ts

```diff
@@ -20,7 +20,17 @@ import { TextDecoder } from "util";
 const PG_FOLDER = ".cobolplugin";
 const PGR_PGM_FILE = "pgm_conf.json";
 const PG_PROC_FILE = "proc_grps.json";
+const DEFAULT_PROCESSOR_GROUP_CONFIG = "[]";
+const DEFAULT_PROCESSOR_CONFIG = '{ "pgms": [] }';
+type WorkspaceConfigEntry =
+  | [string | undefined, string | undefined]
```

Comment:
Would it make sense to cache the decode result object?

Reply:
Thanks ! Yes, will update the PR

Reply:
updated at 5724dd49166a8a7df139c735b2a85af6fc9e32a4